tests/conftest.py

```diff
@@ -13,6 +14,13 @@
 import cmd2
 from cmd2.utils import StdSim
 
+# Python 3.4 require contextlib2 for temporarily redirecting stderr and stdout
```

Comment:
On a vaguely related note, I've been meaning to create an Issue to discuss when we should end support for Python 3.4.  Support for further bug fixes to Python 3.4 ended on March 15, 2019.  We should plan to continue to support it for awhile yet, but should probably end official support for it sometime in 2019 simply to ease the burden on our own maintenance.

Reply:
I created Issue #661 for this